schema/partner.js

```diff
@@ -86,6 +91,24 @@ const PartnerType = new GraphQLObjectType({
           }));
         },
       },
+      artworks: {
+        type: new GraphQLList(Artwork.type),
+        args: {
+          size: {
+            type: GraphQLInt,
+          },
+          sort: ArtworkSorts,
+          exclude: {
+            type: new GraphQLList(GraphQLString),
+          },
+        },
+        resolve: ({ id }, options) => {
+          return gravity(`filter/artworks`, assign({}, options, {
```

Comment:
@mzikherman I understand why you are suggesting not using `/filter/artworks`. Is there an alternative to be able to implement the sort argument? That's why it was initially used. Should the sort just happen on the client side?